src/Custom/Audio/AudioClient.cs

```diff
@@ -229,6 +229,11 @@ public virtual AsyncCollectionResult<StreamingAudioTranscriptionUpdate> Transcri
         Argument.AssertNotNull(audio, nameof(audio));
         Argument.AssertNotNullOrEmpty(audioFilename, nameof(audioFilename));
 
+        if (string.Equals(_model, "whisper-1", StringComparison.OrdinalIgnoreCase))
```

Comment:
We may also want to have this governed by an environment variable so that customers can opt-out if `whisper-1` ever does support it.

Reply:
Thanks! Added support for the environment variable to allow opt-out if needed in the future.